src/sailor/db/elasticsearch.lua

```diff
@@ -0,0 +1,132 @@
+-------------------------------------------------------------------------------------------
+-- elasticsearch.lua: DB module for connecting, querying and serching through elasticsearch
+-- This file is a part of Sailor project
+-- Copyright (c) 2016 Etiene Dalcol <dalcol@etiene.net>
+-- License: MIT
+-- http://sailorproject.org
+-------------------------------------------------------------------------------------------
+
+
+
+local elasticsearch = require "elasticsearch"
+local client = elasticsearch.client()
+local elastic = {}
+
+
+
+
+function elastic.getinfo()
+	local data, err = client:info()
+	if data==nil then
+		return err
+	else
+		return data
+
+	end
+end
+
+function elastic.index(indexq, typeq, idq, body)
+	local data, err = client:index{
+	index = indexq,
+	type = typeq,
+	id = idq,
+	body = body
+}
+	if data==nil then
+		return err
+	else
+		return data
+
+	end
+end
+
+function elastic.get(indexq, typeq, idq)
+	local data, err = client:get{
+  	index = indexq,
+  	type = typeq,
+  	id = idq
+}
+	if data==nil then
+		return err
+	else
+		return data
+	end
+end
```

Comment:
It does not make sense to me as well. is it the way we do with DBs? If not, I would suggest removing the same thing as the request does and let the user handle that in the app. Sometimes the response code is very important to chose what to do next and the current wrapper is doing nothing.

By the way, it is very hard to read the code with different indentation and spaces in each commit. You should decide in one way of writing code and stick to it. Sometimes there is space between '=' sometimes not.